adbg/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: GPL-v2
```

Comment:
Commit message `optee test application: Initial commit`: please, make it according to community standards, mention source and revision.

Reply:
Reworded